### DISCUSSION TITLE
Storybook Docs Code Preview changed between v5 and v6

### DISCUSSION THREAD

**Discussion by TomWija** [storybookjs/storybook#11524] [Help]
Hi everyone, hoping someone might be able to help me out with this. Writing some documentation for a library we built and when I was writing docs using v5, the stories used to have a code preview that showed the entire function being used to achieve the story like below:
![image](https://user-images.githubusercontent.com/7304281/87312411-6f92cc80-c518-11ea-8f8c-0a6a74daaaef.png)

We decided to use addon/controls as well, and for compatibility, we updated our docs addon to v6.0.0 and now the code previews only show the returned react component like so:

![image](https://user-images.githubusercontent.com/7304281/87312548-9ea93e00-c518-11ea-88bc-2ad345531bd5.png)

This is actually useless for what we wanted to achieve with writing our documentation in storybook and can't actually just be copied and pasted by our users to get started with. I was wondering if there was a way to make the v6 code preview function like v5?

**Comment by shilman** [comment]
Will add as #11526

**Comment by TomWija** [comment]
Still having issues with this as of #11526 being closed: 

https://github.com/storybookjs/storybook/issues/11526#issuecomment-660926138

Where source code used to be found with no problems, it's now just saying "no code available"

**Reply by shilman** [reply]
@TomWija do you have a repro I can look at?

**Reply by TomWija** [reply]
Hey, I'm not able to give you access to the main repo, but I put together this one that replicates the problem https://github.com/TomWija/storybook-source-test
